tests/specializations/stores/test_bluebrain_nexus.py

```diff
@@ -119,16 +120,38 @@ def registered_person(person, store_metadata_value):
     return person
 
 
+@pytest.fixture
+def production_configuration():
+    return load_yaml_from_file(
+        full_path_relative_to_root("./examples/notebooks/use-cases/prod-forge-nexus.yml")
+    )
+
+
+@pytest.fixture
+def store_config(production_configuration):
+    return production_configuration["Store"]
+
+
 @pytest.fixture
 @mock.patch("nexussdk.projects.fetch", return_value=NEXUS_PROJECT_CONTEXT)
 @mock.patch("nexussdk.resources.fetch", side_effect=nexussdk.HTTPError("404"))
-def nexus_store(context_project_patch, metadata_context_patch):
+def nexus_store(context_project_patch, metadata_context_patch, store_config):
+
+    store_config_cp = copy.deepcopy(store_config)
+
+    for key in ["endpoint", "model", "bucket", "file_resource_mapping", "token"]:
+        if key in store_config_cp:
+            store_config_cp.pop(key)
+
+    store_config.pop("endpoint")
```

Comment:
WHy pop endpoint here ?

Reply:
My mistake, I should remove it as it's done in the first loop.